packages/app/src/specs/SpecsList.vue

```diff
@@ -448,4 +464,9 @@ function onRunAllSpecs (directory: string) {
 .spec-list-container {
   height: calc(100% - 112px)
 }
+
+/** For run all specs group hover to work */
```

Comment:
Can use `group` and `group-hover`. https://tailwindcss.com/docs/hover-focus-and-other-states#group-hover

I will push the change and link.